src/components/ProgramRecordSendModal/SendLearnerRecordModal.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ModalDialog, Form, SelectableBox, ActionRow, Button,
+} from '@edx/paragon';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+
+import sendRecords from './data/service';
+
+const SendLearnerRecordModal = ({
+  isOpen, toggleSendRecordModal, creditPathways, uuid, username, setSendRecord,
+}) => {
+  const [selectedPathways, setSelectedPathways] = useState([]);
+
+  const handleSendRecords = async () => {
+    const pathwaysToSendRecords = creditPathways.filter(pathway => selectedPathways.includes(pathway.name));
+
+    const orgs = {
+      sendRecordSuccessOrgs: [],
+      sendRecordFailureOrgs: [],
+    };
+
+    await Promise.all(pathwaysToSendRecords.map(pathway => sendRecords(uuid, username, pathway.id)
+      .then(response => {
+        if (response.status >= 200 && response.status < 300) {
+          orgs.sendRecordSuccessOrgs.push(pathway);
+        } else {
+          orgs.sendRecordFailureOrgs.push(pathway);
+        }
+      })
+      .catch(error => {
+        orgs.sendRecordFailureOrgs.push(pathway);
+        const errorMessage = (`Error: Could not send ${pathway.name} record: ${error.message}`);
+        logError(errorMessage);
+      })));
+
+    setSendRecord(prev => ({
+      ...prev,
+      ...orgs,
+    }));
+
+    toggleSendRecordModal();
+  };
+
+  const handleCheckboxChange = (e) => {
+    e.persist();
+
+    if (e.target.checked) {
+      setSelectedPathways(prev => ([
+        ...prev,
+        e.target.value,
+      ]));
+    } else {
+      setSelectedPathways(prev => ([
+        ...prev.filter(pathway => pathway !== e.target.value),
+      ]));
+    }
+  };
+
+  return (
+    <>
+      <ModalDialog
+        title="Send Program Record"
```

Comment:
This looks like a hardcoded english string. Can we resource this?

Reply:
The `title` attribute corresponds to the `aria-label` of the element. Do we need to translate those as well?

Reply:
Ah, ok! I don't think we've translated `aria-label`'s in the past. This is probably fine to ignore.